tests/atest/rules/too-many-arguments/expected_output.txt

```diff
@@ -0,0 +1 @@
+${rules_dir}${\}test.robot:15:0 [W] 0507 Keyword has too many arguments (6/5). Reduce number of arguments or split them into separate lines
```

Comment:
Does splitting arguments to separate lines prevent the warning from showing up? If not - we need to remove this part from the warning message.

Reply:
It does not - we  should rework it. I will note it down as separate issue